xml/docu_styleguide.manage.xml

```diff
@@ -195,6 +195,14 @@
      </callout>
     </calloutlist>
    </example>
+   <para>If an entity is placed at the beginning of a phrase or title, its resolved 
+   form is usually lowercase (unless specified in uppercase, for example, for product names).
+   To have it capitalized, use the <tag class="emptytag">phrase role="style:sentencecase"</tag> element. 
+   For example:</para>
+  <screen>&lt;phrase role="style:sentencecase"&gt;<tag class="genentity">ulp</tag>&lt;/phrase&gt;</screen>
+  <para>The entity <tag class="genentity">ulp</tag> expands into <quote>User space live patching</quote>.</para>
+  <para>Never add this tag to the content within <tag class="emptytag">command</tag> and <tag class="emptytag">systemitem</tag> elements.
```

Comment:
Maybe add a reason to this? Something like:

```suggestion
  <para>Never add this tag to the content within <tag class="emptytag">command</tag> and <tag class="emptytag">systemitem</tag> elements.
  Capitalizing the content of those elements makes the content ambiguous.
```

Maybe there is a better way to express it, but I hope you get the idea. Or just leave it out.